modules/calc/src/main/java/com/opengamma/strata/calc/runner/CalculationTasks.java

```diff
@@ -102,28 +102,27 @@ public static CalculationTasks of(
    * @param rowIndex  the row index of the value in the results grid
    * @param columnIndex  the column index of the value in the results grid
    * @param target  the target for which the measure will be calculated
-   * @param column  the column for which the value is calculated
+   * @param effectiveColumn  the effective column with all rules merged
```

Comment:
How about "the effective column containing the column rules merged with the default rules"?